pyigm/cgm/cgm.py

```diff
@@ -160,7 +160,7 @@ def __init__(self, galaxy, igm_sys, cosmo=None, name=None, rho=None, PA=None,
         # Impact parameter and PA
         if rho is None:
             rho, iang = calc_cgm_rho(galaxy, igm_sys, self.cosmo, ang_sep=ang_sep,
-                                 correct_lowz=correct_lowz, **kwargs)
+                                     correct_lowz=correct_lowz, **kwargs)
```

Comment:
No tabs, please.

Reply:
I don't understand.  Could you elaborate?

Reply:
Will move the correct_lowz... stuff back to the left in case that's what you're talking about.